src/util/taproot.rs

```diff
@@ -142,6 +143,8 @@ pub const TAPROOT_LEAF_MASK: u8 = 0xfe;
 /// Tapscript leaf version
 // https://github.com/bitcoin/bitcoin/blob/e826b22da252e0599c61d21c98ff89f366b3120f/src/script/interpreter.h#L226
 pub const TAPROOT_LEAF_TAPSCRIPT: u8 = 0xc0;
+/// Taproot annex prefix
+pub const TAPROOT_ANNEX_PREFIX: u8 = 0x50;
```

Comment:
we may want to use this constant also in https://github.com/rust-bitcoin/rust-bitcoin/blob/master/src/util/sighash.rs#L728

Reply:
Did as an additional commit